config/config_export_variables.go

```diff
@@ -0,0 +1,78 @@
+package config
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+	"strings"
+
+	"github.com/coveooss/gotemplate/v3/hcl"
+	"github.com/hashicorp/terraform/configs"
+	"gopkg.in/yaml.v2"
+)
+
+type ExportVariables struct {
+	Path   string `hcl:"path"`
+	Format string `hcl:"format,optional"`
+}
+
+// SaveVariables saves variables in paths defined in the export_variables blocks
```

Comment:
maybe should be called `saveExportedVariablesToFile`?

Reply:
Renamed `ExportVariables` to match the actual config name